### ISSUE TITLE
User reviews tiles take the size of the longest review

### ISSUE THREAD

**Issue by forteller** [mjakeman/extension-manager#244]
If one review is really long, all the tiles take up the same large space

![Screenshot from 2022-08-24 13-17-22](https://user-images.githubusercontent.com/343561/186405566-164ca74e-914c-4c61-b480-d4b00c431b48.png)

Thanks! :)

**Comment by mjakeman** [comment]
Yes, this is something I couldn't quite figure out for 0.3, although I've never seen it look quite that bad 😅

I think this will require some custom widgetry to fix, some kind of 'waterfall' tile layout. I'll mark this as 'help wanted' for now but it's on my backlog to fix before 1.0
